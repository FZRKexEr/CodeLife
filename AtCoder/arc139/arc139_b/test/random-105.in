100
36 4 2 4 4 5
94 11 19 10 10 1
25 12 16 16 4 15
5 18 14 16 7 7
92 16 9 1 5 16
59 19 3 20 8 5
60 10 20 12 13 16
20 8 1 20 3 5
37 18 13 15 19 9
65 2 5 15 10 15
89 3 7 10 12 20
38 4 1 4 15 20
29 13 15 19 16 16
75 3 10 12 15 12
63 1 11 12 3 2
33 18 17 13 9 1
52 20 16 9 14 11
37 6 6 6 9 18
1 18 10 17 19 14
65 20 17 16 9 7
7 20 2 13 10 13
40 4 13 20 7 11
61 6 10 19 2 2
91 5 10 5 18 7
64 6 18 8 13 8
94 9 17 8 19 19
24 20 13 12 9 14
83 18 2 13 18 19
91 15 20 4 3 10
83 16 8 12 7 15
65 8 15 13 16 13
2 11 8 20 13 12
84 16 5 1 8 9
54 14 6 5 6 1
17 5 17 5 13 17
47 20 13 8 15 16
51 16 1 14 1 7
39 10 18 7 2 7
97 4 7 7 8 20
46 11 13 12 10 5
2 8 14 13 1 11
67 5 6 18 2 8
66 19 10 2 16 18
14 19 14 10 9 15
42 13 19 15 12 18
31 12 11 6 8 5
21 9 19 15 8 14
25 9 12 1 14 11
17 13 17 18 20 1
21 12 16 16 9 14
32 12 16 4 2 8
71 12 12 14 17 2
26 4 17 19 11 10
36 1 2 7 18 10
36 20 7 20 16 4
27 6 1 9 14 9
68 2 10 8 9 12
68 5 9 18 6 12
44 10 10 1 4 18
25 14 1 7 19 18
7 17 5 8 9 2
99 1 14 5 14 2
10 10 13 7 6 11
40 4 11 4 19 13
21 5 9 6 3 8
12 13 16 3 5 4
45 12 7 8 3 5
86 12 4 1 18 15
48 12 15 10 9 19
78 1 10 18 15 9
8 12 16 16 12 18
33 2 15 14 18 11
16 14 7 7 5 17
53 8 10 12 6 15
38 4 17 18 16 12
73 3 14 5 4 17
78 15 2 14 4 8
18 7 16 16 2 4
64 14 6 3 14 13
51 20 16 4 4 4
13 3 6 2 6 5
24 12 16 7 5 19
93 20 12 4 9 4
23 20 17 3 18 2
12 14 5 3 12 3
100 10 17 14 11 2
20 4 13 17 16 12
16 12 9 20 11 15
27 18 17 9 9 4
72 1 18 11 10 12
28 6 18 16 13 10
23 13 13 4 14 8
31 14 19 10 1 7
37 3 17 20 6 12
95 11 2 3 20 4
33 17 20 20 2 2
94 9 7 19 11 7
36 18 5 14 8 3
95 15 4 6 10 8
18 14 14 1 10 14
