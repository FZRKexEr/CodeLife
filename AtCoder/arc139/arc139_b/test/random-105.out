36
62
24
80
37
51
48
26
53
195
268
143
51
123
34
204
36
60
17
107
52
53
31
68
69
114
57
180
22
71
117
40
44
9
17
55
24
18
199
35
26
38
66
15
81
58
44
23
1
94
16
164
51
180
40
83
108
84
20
96
18
19
6
67
18
36
21
86
36
183
128
54
19
66
40
89
42
36
75
24
12
10
37
20
12
69
79
91
90
48
87
48
32
30
175
262
104
16
74
14
